grpcx/server.go

```diff
@@ -56,22 +58,67 @@ func WithRequestResponseLogs(l logx.Logger) grpc.UnaryServerInterceptor {
 
 // WithErrorLogs returns a gRPC interceptor for unary calls that instrument requests
 // with logs for the errors.
-func WithErrorLogs(l logx.Logger) grpc.UnaryServerInterceptor {
+func WithErrorLogs(l logx.Logger, options ...WithErrorLogsOption) grpc.UnaryServerInterceptor {
+	var logOptions = &withErrorLogsOptions{
+		debugLevelCodes: []codes.Code{},
+		discardedCodes:  []codes.Code{},
```

Comment:
i don't even think you need to initialize them.

Reply:
you're right!